README.md

```diff
@@ -194,3 +194,34 @@ const result = await client.models.transfer.confirmMany(
   </body>
 </html>
 ```
+
+## MFA (Multi-Factor Authentication)
+
+Accessing Kontist banking APIs require Multi-Factor Authentication (MFA).
+
+MFA is available once you have installed the Kontist application and paired your device in it.
+
+The following steps are necessary to complete the MFA procedure:
+1. initiate the procedure by creating a challenge (Kontist SDK exposes a method to do that)
+2. click the push notification you received on your phone, it will open the Kontist application
+3. login (if applicable) and confirm the MFA by clicking on the corresponding button
+
+Kontist SDK exposes a method to initiate the MFA flow after you successfully received the initial access token:
+
+```typescript
```

Comment:
I wasn't sure if I should have included these examples in the regular `Usage` paragraphs, or as a standalone here 🤔

Reply:
We should add that to https://kontist.dev/sdk as well.

Reply:
Yeah I was planning to do it but I wanted first to get some feedback on it here so that I don't have to make changes in both places 🙂